perception/traffic_light_ssd_fine_detector/lib/src/plugins/grid_priors.cpp

```diff
@@ -0,0 +1,240 @@
+// Copyright 2023 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#include "grid_priors.hpp"
+
+#include "grid_priors_kernel.hpp"
+#include "trt_serialize.hpp"
+
+#include <assert.h>
+
+namespace ssd
+{
+namespace
+{
+static const char * PLUGIN_VERSION{"1"};
+static const char * PLUGIN_NAME{"GridPriorsTRT"};
+}  // namespace
+
+GridPriors::GridPriors(const std::string & name, const nvinfer1::Dims & stride)
+: mLayerName(name), mStride(stride)
+{
+}
+
+GridPriors::GridPriors(const std::string & name, const void * data, size_t length)
+: mLayerName(name)
+{
+  deserialize_value(&data, &length, &mStride);
+}
+
+// IPluginV2 methods
+const char * GridPriors::getPluginVersion() const noexcept
+{
+  return PLUGIN_VERSION;
+}
+
+const char * GridPriors::getPluginType() const noexcept
+{
+  return PLUGIN_NAME;
+}
+
+int GridPriors::initialize() noexcept
+{
+  return STATUS_SUCCESS;
+}
+
+void GridPriors::terminate() noexcept
+{
+}
+
+void GridPriors::destroy() noexcept
+{
+  delete this;
+}
+
+void GridPriors::setPluginNamespace(const char * pluginNamespace) noexcept
+{
+  mNamespace = pluginNamespace;
+}
+
+const char * GridPriors::getPluginNamespace() const noexcept
+{
+  return mNamespace.c_str();
+}
+
+int GridPriors::getNbOutputs() const noexcept
```

Comment:
How about this?
```suggestion
int GridPriors::getNumberOfOutputs() const noexcept
```
```suggestion
int GridPriors::getNumOfOutputs() const noexcept
```

```suggestion
int GridPriors::getOutputNum() const noexcept
```

Reply:
@yukkysaito It sounds better and I would love to change:cry:. However, this member function is overrided from base class. Therefore I can not rename this...